### PR TITLE
Change run instructions to run each input json three times

### PR DESCRIPTION
**Pull request by hannahbaumann**
## Input submission checklist

* [ ] created a directory for each system based on the [template directory][templates]
* [ ] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [ ] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [ ] added a PDB file with the protein named `protein.pdb`
* [ ] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [ ] kept cystallographic waters and metals in the PDB file
* [ ] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [ ] copied over the ligands SDF file to a file named `ligands.sdf`
* [ ] copied over the edges CSV file to a file named `edges.csv`

## Licensing agreement
* [ ] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

**Comment by hannahbaumann** [comment]
@mikemhenry : Is this similar to what you've been doing for the PLB QA? If not, could you please change the run instructions?

**Comment by hannahbaumann** [comment]
Should in addition to the run instructions also provide an actual run submission script example @IAlibay ?

File: docs/source/public/overview.rst
Comment (IAlibay) [inline docs/source/public/overview.rst:1]: > You can find additional information and examples on how to run simulations of the entire network in the "Running the simulations" section of our `CLI tutorial

Do you think we should change this to be a bit clearer that we're doing this a slightly atypical manner? Maybe something like "this is a slightly modified approach to our CLI Tutorial, allowing for execution of a single task per HPC job"?

File: docs/source/public/overview.rst
Comment (IAlibay) [inline docs/source/public/overview.rst:187]: We should say somewhere that we are happy to help folks adapt this script to their own internal needs as necessary.

**Review by IAlibay — CHANGES_REQUESTED** [review]

File: docs/source/public/overview.rst
Comment (IAlibay) [inline docs/source/public/overview.rst:191]:
```suggestion
Please reach out to the openfe team if you have any questions on how to adapt this script to your internal needs, we would be happy to assist with this.
```
Slightly less colloquial :)

**Review by IAlibay — APPROVED** [review]
Just the one thing - I'll self merge

**Comment by IAlibay** [comment]
I'll let @mikemhenry review before we merge the PR.

File: docs/source/public/overview.rst
Comment (RiesBen) [inline docs/source/public/overview.rst:181]: I think the line above needs to be change to this: 

```bash
 for file in network_setup/transformations/*.json; do
```

not sure if the rest of the rel-paths work then?

**Review by RiesBen — CHANGES_REQUESTED** [review]
I think I found something :)

File: docs/source/public/overview.rst
Comment (IAlibay) [reply docs/source/public/overview.rst:181]: I don't have a strong view here - possibly putting thing sin a "transformations" folder might be useful so the top level directory is cleaner. That being said, I consider this to be reasonably low priority so I'm leaving it up to you @hannahbaumann

Comment (hannahbaumann) [reply docs/source/public/overview.rst:181]: Ok! I can change it, just to be more similar to the regular CLI output, so that people could reuse the same run script if they used the regular CLI to create their networks.

Comment (hannahbaumann) [reply docs/source/public/overview.rst:181]: Edit: I just realized that I would also have to change the run script to create the extra directory and I'm not sure if it's worth it since besides the .jsons files there's only the ligand_network.graphml file in the folder. So maybe we do leave it as is, unless others think it would be beneficial to be consistent @IAlibay @RiesBen

Comment (IAlibay) [reply docs/source/public/overview.rst:181]: If it means not adding any last minute work, I am more than happy to go with the current state of things @hannahbaumann !

Comment (RiesBen) [reply docs/source/public/overview.rst:181]: Hi, I think it would be great if our folder structure follows this here: 
https://docs.openfree.energy/en/stable/tutorials/rbfe_cli_tutorial.html

in my understanding the final folder structure, after full execution should look like this:

some_benchmark_folder

--network_setup
-----ligand_*json
-----transformations
-------*jsons

--results
----fun1
------shared_*
--------*nc
-----fun1.json
----fun2
------shared_*
--------*nc
-----fun2.json
---- ...

--final_results.tsv

Comment (RiesBen) [reply docs/source/public/overview.rst:181]: will transfer to an issue

**Review by RiesBen — APPROVED** [review]
Ok, 
i checked the PR again, my comment of the folder structure was transferred to an issue.

I like the PR :)

**Pull request merged**
